pression/compressor.cpp

```diff
@@ -62,7 +62,7 @@ class Compressor : public PluginInstance
         if( name <= EQ_COMPRESSOR_NONE )
             return true;
 
-        plugin = registry.findPlugin( name );
+        plugin = pression::PluginRegistry::getInstance().findPlugin( name );
         LBASSERT( plugin );
         if( !plugin )
             return false;
```

Comment:
1) This is the legacy API. I'm not going to change it in this PR.
2) Changing the return value would change its semantics.